spec/features/admin_deletes_all_comments_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'spec_helper'
+feature 'As an admin, I want to be able to delete all reviews', %Q{
```

Comment:
Use `%` instead of `%Q`.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Use `%Q` only for strings that contain both single quotes and double quotes, or for dynamic strings that contain double quotes.